### ISSUE TITLE
"Function called with incorrect arguments" error from task_run_all

### ISSUE THREAD

**Issue by Manu343726** [toeb/cmakepp#120]
See http://stackoverflow.com/q/31224618/1609356

I assume you are currently busy and out of cmakepp stuff. I just want a hint, what could be the reason for this kind of internal error?

Thanks in advance.

**Comment by toeb** [comment]
Yeah sorry.  I can't seem to get back to it atm just started a job.    

 in cmakepp you can register tasks in every file which are executed after cmakepp has loaded.  similar to jquery's $(handler) function.  If you never use the task_ functions then it stands to reason that there is some kind of snafu in the task system (which I have reworked in newer versions of cmakepp) .  Else if you use the task system then there might be a problem with those tasks.  

 If its only the tasks that I myself then one could just delete or comment out the lines which are calling task_enqueue (about 3 in the whole of cmakepp) without causing any error. or by commenting out the task_run_all call in cmakepp.cmake line 110 as they are problably not necessary and only pertain to my project&package management functionality.


If that doesnt work here are some further thoughts on the matter:

What I find peculiar is that his path is c:\users\tobi... is his username Tobi?  Because mine is too and I am questioning myself if the path is hard coded.. (but it shouldn't matter)

also what revision is he using?  

another question:  are the tasks being run in parallel or in sequence?  This could indicate  a problem with the temporary file that cmake uses (its the same in every instance and is actually used as a kind of register in the eval function - ie the filename never changes)

could you maybe also post a link to 
```
  ../deps/boost/install/build_jobs.cmake:1 (include)
  ../deps/boost/install/install.cmake:2 (include)
  ../deps/biicode/boost/setup.cmake:1 (include)
  ../../CMakeLists.txt:1 (include)
```

There could be a problem with the parsing of the startup tasks
